internal/output/table.go

```diff
@@ -118,9 +118,11 @@ func printSummaryResult(result Result, outputWriter io.Writer, terminalWidth int
 	}
 
 	for _, ecosystem := range result.Ecosystems {
-		fmt.Fprintln(outputWriter, ecosystem.Name)
-
 		for _, source := range ecosystem.Sources {
+			if source.PackageTypeCount.Regular == 0 {
+				break
```

Comment:
one ecosystem contains multiple sources, so we need to do `continue` here instead of `break` the entire for loop.